breacharbiter.go

```diff
@@ -141,40 +147,59 @@ func (b *breachArbiter) Start() error {
 
 	brarLog.Tracef("Starting breach arbiter")
 
-	// We load all pending retributions from the database and
-	// deterministically reconstruct a channel close summary for each. In
-	// the event that a channel is still open after being breached, we can
-	// use the close summary to reinitiate a channel close so that the
-	// breach is reflected in channeldb.
+	// TODO(roasbeef): instead use closure height of channel
+	_, currentHeight, err := b.cfg.ChainIO.GetBestBlock()
```

Comment:
why moved here?

Reply:
We want to grab the current height ASAP to ensure we don't skip blocks while startup/recovery is happening, can be improved later w/ registering at target height/hash